setup.py

```diff
@@ -26,12 +26,19 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 import io
+from pip._internal.req import parse_requirements
 
 from setuptools import find_packages, setup
 
 with io.open('README.rst', 'rt', encoding='utf8') as f:
     readme = f.read()
 
+
+def load_requirements(fname):
+    reqs = parse_requirements(fname, session='build')
+    return [str(ir.requirement) for ir in reqs]
+
+
 setup(
     name='servicex_app',
     version='1.0.0-RC.3',
```

Comment:
Should the version in `setup.py` be changed on the release branch? I'm not sure that it matters..